src/component/inspect-request/request-details.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import RequestHeaders from './request-headers.js';
+import FullUrl from '../request/full-url.js';
+
+const {object} = React.PropTypes;
+
+const RequestDetails = (props) => {
+  const {request} = props;
+
+  return <div className="box-body">
+    <section>
+      Request URL:
+      <FullUrl request={request.request} />
+    </section>
+    <section>
+      Request Query Params:
+      <RequestHeaders headers={request.request.query} />
```

Comment:
I was thinking of that, hadn't thought of the best approach yet. I was thinking of adding a label prop to `RequestHeaders`, so it can be handled in there.

```jsx
<RequestHeaders label="Request Query Params" headers={request.request.query} />
```

That's one thought, anyway. But definitely agree that we shouldn't show an empty section.